src/sdk/namespace_cache.js

```diff
@@ -561,6 +568,16 @@ class NamespaceCache {
 
             if (cache_ok) {
                 assert.strictEqual(hub_res.value.etag, cache_res.value.etag);
+
+                const update_params = _.pick(_.defaults(
+                    { bucket: this.namespace_nb.target_bucket }, params), 'bucket', 'key', 'last_modified_time');
+                update_params.last_modified_time = (new Date(hub_res.value.last_modified_time)).getTime();
+                try {
+                    await object_sdk.rpc_client.object.update_object_md(update_params);
```

Comment:
We need to decide what order we prefer:
1. Complete the upload to an object, then update_object_md - opens up a small window where the last_modified_time can be returned from the cache incorrectly.
2. Update_*upload*_md, and then complete upload - need to add the update_upload_md() api for that.
Need to decide...

Reply:
@guymguym I pushed commit proposing solution addressing the issue in updating last_modified_time. Could you take a look at it? If good, we can apply the same to upload_object() calls in the read scenarios. 
We also need to handle the partial object creation case, propose to add last_modified_time in create_object_upload API for partial object (complete upload) scenario. So if partial object is created, last_modified_time needs to be provided just like etag. Thought?